trace/opencensus/trace/samplers/fixed_rate.py

```diff
@@ -23,7 +23,7 @@ class FixedRateSampler(Sampler):
     :type rate: float
     :param rate: The rate of sampling.
     """
-    def __init__(self, rate):
+    def __init__(self, rate=0.5):
```

Comment:
Done. Yeah I remember there was a discussion about this. We can always change the default when we have a conclusion later.